burrow/setup.py

```diff
@@ -0,0 +1,51 @@
+from codecs import open  # To use a consistent encoding
+from os import path
+
+from setuptools import setup
+
+HERE = path.dirname(path.abspath(__file__))
+
+# Get version info
+ABOUT = {}
+with open(path.join(HERE, 'datadog_checks', 'burrow', '__about__.py')) as f:
+    exec(f.read(), ABOUT)
+
+# Get the long description from the README file
+with open(path.join(HERE, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
+
+CHECKS_BASE_REQ = 'datadog-checks-base>=11.0.0'
```

Comment:
This is not going to work with agent 5, if you want to use this with agent 5 you need to specify an older checks-base version